README.rst

```diff
@@ -13,8 +13,11 @@ PyGMT
     :alt: Travis CI build status
     :target: https://travis-ci.org/GenericMappingTools/pygmt
 .. image:: https://github.com/GenericMappingTools/pygmt/workflows/Tests/badge.svg
-    :alt: GitHub Actions build status
-    :target: https://github.com/GenericMappingTools/pygmt/actions
+    :alt: GitHub Actions Tests status
+    :target: https://github.com/GenericMappingTools/pygmt/actions?query=workflow%3ATests
+.. image:: https://github.com/GenericMappingTools/pygmt/workflows/GMT%20Master%20Tests/badge.svg
+    :alt: GitHub Actions GMT Master Tests status
+    :target: https://github.com/GenericMappingTools/pygmt/actions?query=workflow%3A"GMT+Master+Tests"
```

Comment:
> I think we should move the "testing GMT master branch" into a separate file. Sometimes it's impossible to make all the tests for the current release and master branch. Then we will always see a failing badge in the README file.

Ok, I've moved it into a separate file and added a separate badge. It would be nice if we could have separate badges for different jobs in a workflow but it doesn't look to be possible yet according to https://github.community/t/separate-workflow-badges-when-using-matrix-testing-possible/16708.